### DISCUSSION TITLE
Paperless-ngx LXC Export Documents

### DISCUSSION THREAD

**Discussion by Maagi93** [community-scripts/ProxmoxVE#1084] [Q&A]
Hi, I installed Paperless-ngx LXC with the helper script https://community-scripts.github.io/ProxmoxVE/scripts?id=paperless-ngx. How can I get my documents for an export? How can I start the document exporter?

**Comment by Rookman2022** [comment]
I would agree with the question. I would like to move from the LXC with the helper script installed Paperless to a Docker variant, but I have not yet found a way to export the data or database and import it back into the new system. The exporter available in Paperless does not work in the LXC container installed with the helper script. I cannot import a database exported with the Adminer into the Docker container.

**Comment by Ullulaki** [comment]
Had the same problem some time ago. Here are my notes I made:

**Export**
1. switch to Folder:
cd /opt/paperless/src

2. export the files with the following command:
python3 manage.py document_exporter /destination path for export/

in my example:
python3 manage.py document_exporter /home/paperless-export

3. now zip the folder:
tar -czf paperless-export.zip /home/paperless-export/

***

**Import**
1. move before created zip-file to destination

2. extract zip:
tar -xzf paperless-export.zip /destination path for export/

in my example:
tar -xzf paperless-export.zip /home/paperless-export/

3. now start the import in the paperless/src/-directory:
switch to directory:
cd /opt/paperless/src

enter import command:
python3 manage.py document_importer /home/paperless-export

**Reply by Rookman2022** [reply]
You are my hero !!!
Thank you very much, sometimes everything can be so easy.

A small error is in point 3 when exporting in the tar command the target file must be named.
So: tar -czf paperless-export.zip /home/paperless-export/

But that helped me so much, thank you very much :-)

**Reply by Ullulaki** [reply]
Glad I was able to help you.
And thanks for the info, just edited the post 😎

**Comment by cemkur** [comment]
Hi there, I have the same issue (Exporting from LXC and moving Docker VM). I am on v 2.20.1. I followed your instruction but the command

python3 manage.py document_exporter /home/abcd

resulted in

Traceback (most recent call last):
  File "/opt/paperless/src/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.11/dist-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.11/dist-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/usr/local/lib/python3.11/dist-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.11/dist-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/django/apps/config.py", line 193, in create
    import_module(entry)
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1142, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'treenode'

I am not sure where to go from here. Any advice?

Thank you

**Reply by MickLesk** [reply]
uv run python3 manage....

**Comment by cemkur** [comment]
Worked great. When importing the data, inside the docker container, I didn't need to use the "uv run" part. Just "python3 manage.py document_importer /exported_folder" sufficed. I just needed to map the exported_folder to the container in the compose file.
Thank you.

**Reply by MickLesk** [reply]
we dont have docker

**Comment by Gugulator** [comment]
Trying to move database from one LXC to another (don't ask why 😊)
Source container was created from TTECK's script.
Everything successfully created and copied from source.
When entering on target LXC to  `/opt/paperless/src`
And running `python3 manage.py document_importer /home/exported/`
I'm getting following error:
`Traceback (most recent call last):
  File "/opt/paperless/src/manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'`

As it looks like, there are many other modules missing.
Is it possible to solve? Maybe something missed in  installation script due to newest versions of Paperless-ngx?

**Reply by MickLesk** [reply]
1 Post above your is the Solution... Like in many other Posts for paperless.

**Reply by Gugulator** [reply]
sorry, i'm a total noob in a things belongs to python commands. it took a time to understand to run it in that way `uv run python3 manage.py document_importer /root/exported/`
but yep, it works, thanks!
